pages/data-design/how-to-add-a-group-to-an-event.mdx

```diff
@@ -0,0 +1,31 @@
+---
+layout: 'guide'
+title: 'How to add a Group to an Event'
+---
+
+Avo is equipped with a familiar user interface to define and maintain Group Types and their properties and updating Group Properties is as easy as adding an action to your Event.
+
+## Step 1 - Add the Update Group action
+
+Go to the event which you wish to send the Group Properties with and choose the `Update Group` action. This action allows you to link users with Groups and/or update Group Properties.
+
+<img
+  src={require('../../images/data-design/add-group-action.png')}
+  alt="How to add the Update Group action"
+/>
+
+## Step 2 - Add Groups and Group Properties
+
+To link users with Groups click `+ Add Group Type` and choose the Group Type you wish to associate the user with. All subsequent calls will link the user to this Group.
+
+<img
+  src={require('../../images/data-design/add-group.png')}
+  alt="How to add a Group"
+/>
+
+To update Group Properties click `+ Add Group Properties` to add as many properties as you'd like from your groups. Avo will manage each separate call respectively. Note that Segment will always link a user with a Group when updating Group Properties.
```

Comment:
Can probably mention that Segment does not support log event with group, that's why Segment will always link a user with a Group when updating Group Properties.